### DISCUSSION TITLE
When i try to keep both Protractor and WDIO in same project getting error related to typescript

### DISCUSSION THREAD

**Discussion by santosjister** [webdriverio/codemod#66] [Migration Issue]
Hi Team,

I am trying to integrate the WDIO into my existing protractor automation project, post the config file migration i got both wdio.conf.ts file and i reverted the Protractor.conf.ts file, now the protractor scripts are working fine however, when i try to run the WDIO  script i am getting 'SyntaxError: Cannot use import statement outside a module' error. I tried different options but still couldn't get out of this error and make WDIO working. 

**Note:** In the same machine i have created sample WDIO project which is working fine, i have manually moved those files to my protractor project and installed all required dependencies to make it work but no luck after integration. The important point is it works fine independently.
I have cross checked the package.json file, tsconfig.json and wdio.conf.ts file between the independently working project and post merge to protractor, all required dependencies are matched.

Could you please help me on this?

**Comment by christian-bromann** [comment]
Do you have [`autoCompileOpts`](https://webdriver.io/docs/options#autocompileopts) set in your `wdio.conf.ts`? I recommend to bootstrap a fresh TS project using `npm init wdio` and see the differences between both projects.

**Comment by santosjister** [comment]
Hi Chris,
I have 'autoCompile' set as 'true' in my 'wdio.conf.ts' file. I am already comparing with the fresh project created in WDIO using 'npm init wdio'. Thats were i coudn't find issue.
